api/config/custom-environment-variables.json

```diff
@@ -29,5 +29,11 @@
     "sizeLimit": "FILE_SIZE_LIMIT",
     "storagePath": "FILE_STORAGE_PATH"
   },
-  "geolocation": {"gmapsApiKey": "GMAPS_API_KEY" }
+  "geolocation": {"gmapsApiKey": "GMAPS_API_KEY" },
+  "password": {
+    "minLength": "PASSWORD_MIN_LENGTH",
+    "includeNumerics": "PASSWORD_INCLUDE_NUMERICS",
+    "includeUpperCase": "PASSWORD_INCLUDE_UPPER_CASE",
+    "includeSpecialCharacters": "PASSWORD_INCLUDE_SPECIAL_CHARACTERS"
+  }
```

Comment:
I think this env vars should be added also to the docker-compose setup for team members running this on docker.

Also it would be great adding them to the dotenv skeleton (.env.default) so people can notice there are new env vars

Reply:
we have reasonable defaults already, so no need to add these to the docker-compose or .env skeleton. But I do think we should have a list documenting all supported config env vars. Maybe add a section to the README documenting the vars you just added.